ersilia/cli/commands/delete.py

```diff
@@ -37,3 +29,41 @@ def delete(model):
                 ),
                 fg="yellow",
             )
+
+    def _delete(md, model_id):
+        md.delete(model_id)
+
+    def _delete_all():
+        """Function to delete all locally available models"""
+        model_catalog = ModelCatalog()
+        catalog_table = model_catalog.local()
+        local_models = catalog_table.data if catalog_table else None
+        if not local_models:
+            echo(":person_tipping_hand: No models are available locally for deletion.", fg="yellow")
+            return
+        deleted_count = 0 
+        for model_row in local_models:
+            model_id = model_row[0]
+            if _delete_model_by_id(model_id):
+                deleted_count += 1
+        echo(":thumbs_up: Completed the deletion of all locally available models!", fg="green")
+        
+
+    # Example usage:
+    # 1. Delete a specific model: ersilia delete {MODEL}
+    # 2. Delete all models: ersilia delete --all
+    @ersilia_cli.command(
+        short_help="Delete model from local computer",
+        help="Delete model from local computer. The BentoML bundle is deleted, as well as the files stored in "
```

Comment:
"Delete model(s) available locally. The BentoML bundle is deleted, as well as the files stored in the EOS directory."